new_mvc/controllers/authController.js

```diff
@@ -41,47 +41,49 @@ exports.validateUser = async (req, res) => {
 
         const [row] = await auth.validateUser(req.body.login);
 
+        const dbUser = row[0];     //  !!!!!
+
         console.log("row:", row);
 
         if (row.length > 0) {
             let passwordEnteredByUser = req.body.pass;
-            let salt = row[0].salt;
```

Comment:
Here and in all pull request - use `const` if variable will not be changed. If it's object - remember that properties could be changed in const